src/components/encounter/encounter-form.component.tsx

```diff
@@ -177,6 +181,10 @@ const EncounterForm: React.FC<EncounterFormProps> = ({
     }
   }, [tempInitialValues]);
 
+  useEffect(() => {
+    setProgramUuid(formJson?.meta?.programs?.uuid);
```

Comment:
Updating this inside a `useEffect` might not be necessary especially if there are no dependencies.